crates/bevy_window/src/window.rs

```diff
@@ -648,25 +644,25 @@ impl WindowResolution {
     /// The window's client area width in logical pixels.
     #[inline]
     pub fn width(&self) -> f32 {
-        self.physical_width() as f32 / self.scale_factor()
+        self.physical_size.to_logical(self.scale_factor).x
```

Comment:
This should call method `scale_factor()`, not access `scale_factor` field. Currently, this wouldn't support `scale_factor_overide`.